### PR TITLE
credits: document some of the debug tools

### PR DESCRIPTION
**Pull request by kemenaran**
Document end credits debug tools, and annotate some of the ending shared timers.

### CODE REVIEW

File: src/code/credits.asm [hunk 1]
```diff
@@ -1008,77 +1008,84 @@ Data_017_49B7::
...
     and  a                                        ; $4AFD: $A7
-    jr   z, .jr_4B04                              ; $4AFE: $28 $04
-
+    jr   z, .decrementwD009End                    ; $4AFE: $28 $04
     dec  a                                        ; $4B00: $3D
     ld   [wD009], a                               ; $4B01: $EA $09 $D0
+.decrementwD009End
 
-.jr_4B04
+    ; Decrement wD0013 every 16 frames
```
Comment (tobiasvl) [inline src/code/credits.asm:1078]:
```suggestion
    ; Decrement wD013 every 16 frames
```

File: src/code/credits.asm [hunk 2]
```diff
@@ -1213,7 +1221,7 @@ label_017_4C22:
     ret                                           ; $4C5A: $C9
 
 IncrementCreditsGameplaySubtype::
-IncrementCreditsGameplaySubtypeAndReturn::
+IncrementCreditsGameplaySubtypeAndreturn::
```
Comment (tobiasvl) [inline src/code/credits.asm:1224]: Why this strange case change, here and on similar labels? It looks like you might have replaced all "Return" with "return" for some reason?

File: src/code/credits.asm [hunk 3]
```diff
@@ -2965,7 +2973,7 @@ CreditsLoadBGMap::
     add  hl, de                                   ; $5B23: $19
     ld   a, [hl+]                                 ; $5B24: $2A
     ld   b, a                                     ; $5B25: $47
-    ; Return bank to restore
+    ; return bank to restore
```
Comment (tobiasvl) [inline src/code/credits.asm:2976]: This makes me think you've done a search/replace, and inadvertently changed the labels too

**Review by tobiasvl — CHANGES_REQUESTED** [review]

File: src/code/credits.asm (hunk 2 above)
Comment (kemenaran) [reply src/code/credits.asm]: Yup, clearly a mistake – I didn't even proof-read this PR myself 🤦 Fixed.

**Review by tobiasvl — APPROVED** [review]

**Pull request merged**